### DISCUSSION TITLE
The Test results tab is always automatically opened

### DISCUSSION THREAD

**Discussion by egor-zalenski** [vitest-dev/vscode#365] [Q&A] [answered]
is it possible that the Test results tab does not automatically open after running any test?
Or to open it only if the test fails.

And is it possible to make it like on the gif in read.me? 

It does not open the whole Test results tab, but only a small area in the editor of a failed test, where the test result is displayed.

How it works now

https://github.com/vitest-dev/vscode/assets/63463140/827a7811-7167-4000-946b-376262454143

**Comment by sheremet-va** [comment, marked as answer]
From https://www.reddit.com/r/vscode/comments/15mg17y/vs_code_now_opens_the_test_results_tab_on_the/:

> The "testing.openTesting" setting can be set to "neverOpen", which seems to make it behave like it did before.

**Reply by egor-zalenski** [reply]
Just what I was looking for. Thank you!
